waspc/cli/src/Wasp/Cli/Command/Db/Start.hs

```diff
@@ -0,0 +1,63 @@
+{-# LANGUAGE TypeApplications #-}
+
+module Wasp.Cli.Command.Db.Start
+  ( start,
+  )
+where
+
+import Control.Monad.IO.Class (liftIO)
+import StrongPath (Abs, Dir, Dir', Path', Rel, fromAbsDir, reldir, (</>))
+import System.Process (callCommand)
+import qualified Wasp.AppSpec as AS
+import qualified Wasp.AppSpec.App as AS.App
+import qualified Wasp.AppSpec.App.Db as AS.App.Db
+import Wasp.Cli.Command (Command)
+import Wasp.Cli.Command.Common
+  ( findWaspProjectRootDirFromCwd,
+    parseWaspFileC,
+  )
+import Wasp.Cli.Command.Message (cliSendMessageC)
+import Wasp.Cli.Common (WaspProjectDir, dotWaspDirInWaspProjectDir)
+import qualified Wasp.Db.Postgres as P
+import qualified Wasp.Message as Msg
+
+start :: Command ()
+start = do
+  waspProjectDir <- findWaspProjectRootDirFromCwd
+  decls <- parseWaspFileC waspProjectDir
+
+  case getDbSystem decls of
+    Nothing -> noteSQLiteDoesntNeedStart
+    Just AS.App.Db.SQLite -> noteSQLiteDoesntNeedStart
+    Just AS.App.Db.PostgreSQL -> startPostgreDevDb waspProjectDir
+  where
+    getDbSystem decls = (snd <$> AS.getApp decls) >>= AS.App.db >>= AS.App.Db.system
+
+    noteSQLiteDoesntNeedStart =
+      cliSendMessageC . Msg.Info $
+        "Nothing to do! You are all good, you are using SQLite which doesn't need to be started."
```

Comment:
Maybe also add, if they wanted to use `PostgreSQL` they should change the `system` field in the `db`` declaration of the Wasp app.

Reply:
Interesting idea but I would not add it here as I think there are better ways for them to learn about PostgreSQL support, I would rather they go through those.